ports/esp32/modutime.c

```diff
@@ -82,6 +84,19 @@ STATIC mp_obj_t time_time(void) {
 }
 MP_DEFINE_CONST_FUN_OBJ_0(time_time_obj, time_time);
 
+STATIC mp_obj_t time_ticks_ms(void) {
+    uint32_t result = esp_timer_impl_get_time() / 1000;
+    return mp_obj_new_int( result & 0xFFFFFFFF );
```

Comment:
Using `mp_obj_new_int()` can lead to heap allocation and so will be slower than using `MP_OBJ_NEW_SMALL_INT()`.  Also, this new formula here (the masking) will no longer work with `ticks_diff()`.